administrate/cache.md

```diff
@@ -15,14 +15,18 @@ keywords:
 [Varnish](https://www.varnish-cache.org/) is a HTTP proxy-cache, which works as a reverse proxy between your application
 and the client. Following rules defined by the user, Varnish will cache the data of an application to reduce the load on its server. We use **Varnish 4**
 
+## Limitations
+
+{{< alert "warning" >}}
+<p>Varnish is only available on PHP >= 5.5 applications. Support for other applications is in discussion.</p> 
+<p>For more information about it, contact us at <support@clever-cloud.com>.</p>
```

Comment:
The `<support@clever-cloud.com>` email address isn't visible anymore because it is interpreted as an HTML tag and it doesn't exist. I've made the following suggestion to fix that

```suggestion
<p>For more information about it, contact us at <a href="mailto:support@clever-cloud.com">support@clever-cloud.com</a>.</p>
```